docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -0,0 +1,136 @@
+The second Function is a message receiver. It subscribes to the given event type and stores the payload.
+
+It shows only one possible use case. You can find plenty of use cases yourself on how to orchestrate your application logic into specialised Functions and benefit from decoupled, re-usable components and event-driven architecture.
+
+## Prerequisites
+
+
+- [Kyma CLI](https://github.com/kyma-project/cli)
+- Kyma installed locally or on a cluster
+
+## Steps
+
+1. Export the `KUBECONFIG` variable:
+   ```bash
+   export KUBECONFIG={KUBECONFIG_PATH}
+   ```
+2. Create two folders in your project - `emitter` and `receiver`.
+
+### Create the emitter Function
+
+1. Go to the `emitter` folder and run Kyma CLI `init` command to initialize a scaffold for your first Function:
+
+   ```bash
+   kyma init function
+    ```
+
+  The `init` command creates these files in your workspace folder:
+
+  - `config.yaml`	with the Function's configuration
+
+      >**NOTE:** See the detailed description of all fields available in the [`config.yaml` file](../../05-technical-reference/svls-06-function-configuration-file.md).
+
+  - `handler.js` with the Function's code and the simple "Hello Serverless" logic
+  
+  - `package.json` with the Function's dependencies
+
+2. In the `config.yaml` file, configure an API Rule to expose your Function to the incoming traffic over HTTP. Provide the subdomain name in the `host` property:
+
+    ```yaml
+    apiRules:
+      - name: incoming-http-trigger
+        service:
+          host: incoming
+        rules:
+          - methods:
+              - GET
+            accessStrategies:
+              - handler: allow
+    ```
+
+3. Provide your Function logic in the `handler.js` file:
+>**NOTE:** In this example, there's no sanitization logic. The `sanitize` Function is just a placeholder.
+
+   ```js
+   const { v4: uuidv4 } = require('uuid');
+   module.exports = {
+       main: function (event, context) {
+           let sanitsedData = sanitise(event.data)
+           var eventOut=event.buildResponseCloudEvent(uuidv4(), "sap.kyma.custom.acme.payload.sanitised.v1", sanitisedData);
+           eventOut.source="kyma"
+           eventOut.specversion="1.0"
+           event.publishCloudEvent(eventOut);
+           console.log(`Payload pushed to sap.kyma.custom.acme.payload.sanitised.v1`,eventOut)
+           return eventOut;
+       }
+   }
+   let sanitise = (data)=>{
+       console.log(`sanitising data...`)
+       console.log(data)
+       return data
+   }
+   ```
+   >**NOTE:** The `sap.kyma.custom.acme.payload.sanitised.v1` is a sample event type declared by the emitter Function when publishing events. You can choose a different one that better suits your use case. Keep in mind the constraints described on the [Event names](https://kyma-project.io/docs/kyma/latest/05-technical-reference/evnt-01-event-names) page. The receiver subscribes to the event type to consume the events.
```

Comment:
```suggestion
   >**NOTE:** The `sap.kyma.custom.acme.payload.sanitised.v1` is a sample event type declared by the emitter Function when publishing events. You can choose a different one that better suits your use case. Keep in mind the constraints described on the [Event names](../../05-technical-reference/evnt-01-event-names.md) page. The receiver subscribes to the event type to consume the events.
```

Reply:
Use relative links to link to the docs within the Kyma project. Otherwise, they will appear as external links on the website:

<img width="745" alt="Screen Shot 2022-01-21 at 11 19 42" src="https://user-images.githubusercontent.com/35192450/150510449-19013b0d-0917-4696-a855-08565b9b6c85.png">